civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -70,3 +70,16 @@ $row-margin-bottom: 15px;
     }
   }
 }
+
+/*
+* This is a TEMP fix to fit the text of input controls like date picker within input box
+* It is hardcoded to 13px for now.
+*/
+.chr_leave-request-modal__input-font-size {
+  font-size: 13px;
+}
+
+.chr_leave-request-modal__span-expiry {
```

Comment:
It is combination of `control-label` and `label` which caused the text to be mis aligned after addition of `uib-datepicker-popup` for expiry date. 
If i use `form-control` in span it gives a border rectangle but aligns it well. is that ok?